app/build.gradle

```diff
@@ -44,9 +44,17 @@ dependencies {
     implementation 'androidx.navigation:navigation-ui:2.5.3'
     implementation 'androidx.annotation:annotation:1.2.0'
     implementation files('libs/tkey-native/tkey-debug.aar')
```

Comment:
This could be changed to use the repository instead of this file, like the tss-client-android below.